File: sample-conference-videochat/sample-conference-videochat/Components/CustomSegues/RootViewControllerSegue.m
//
//  RootViewControllerSegue.m
//  sample-conference-videochat
//
//  Created by Injoit on 12/10/15.
//  Copyright © 2015 QuickBlox Team. All rights reserved.
//

#import "RootViewControllerSegue.h"
#import "AppDelegate.h"

@implementation RootViewControllerSegue

- (void)perform {
    
    AppDelegate *delegate = (AppDelegate *)[UIApplication sharedApplication].delegate;
    delegate.window.rootViewController = self.destinationViewController;
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/Classes/ReplaceSegue/RootViewControllerSegue.swift
//
//  RootViewControllerSegue.swift
//  sample-conference-videochat-swift
//
//  Created by Injoit on 04.10.2018.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import UIKit

class RootViewControllerSegue: UIStoryboardSegue {
    override func perform() {
        let delegate = UIApplication.shared.delegate as? AppDelegate
        delegate?.window?.rootViewController = destination
    }
}
